<!DOCTYPE HTML>
<!--
	Massively by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Jay Shin Lab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<noscript><link rel="stylesheet" href="assets/css/noscript.css" /></noscript>
	</head>
	<body class="is-preload">

		<!-- Wrapper -->
			<div id="wrapper">

				<!-- Header -->
					<header id="header">
						<a href="index.html" class="logo">Jay Shin Lab</a>
					</header>

				<!-- Nav -->
					<nav id="nav">
						<ul class="links">
							<li><a href="index.html">Regulatory Genomics</a></li>
							<li><a href="team.html">Team</a></li>
							<li class="active"><a href="publications.html">Publications</a></li>
							<li><a href="media.html">Media</a></li>
						</ul>
						<ul class="icons">
							<li><a href="https://twitter.com/jayshin" class="icon brands fa-twitter"><span class="label">Twitter</span></a></li>
						</ul>
					</nav>

				<!-- Main -->
					<div id="main">

						<!-- Post -->
							<section class="post">
								<header class="major">
									<h1>Publications</h1><br />
								</header>
										<ol>
											<li>Moody J, Kouno T, Chang JC, Ando Y, Carninci P, <b>Shin JW</b>, Chung-Chau Hon, SCAFE: a software suite for analysis of transcribed cis-regulatory elements in single cells, <b>Bioinformatics</b>, Volume 38, Issue 22, 15 November 2022, Pages 5126–5128. <a href="https://doi.org/10.1093/bioinformatics/btac644"> https://doi.org/10.1093/bioinformatics/btac644</a>.</li>
											<li>Ducoli L, Agrawal S, Sibler E, Kouno T, Tacconi C, Hon CC, Berger SD, Müllhaupt D, He Y, Kim J, D’Addio M, Dieterich LC, Carninci P, de Hoon MJL, <b>Shin JW</b>, Detmar M#. LETR1 is a lymphatic endothelial-specific lncRNA that governs cell proliferation and migration through KLF4 and SEMA3C. <b>Nat. Commun</b> 12: 925, 2021 <a href="https://doi.org/10.1038/s41467-021-21217-0"> https://doi.org/10.1038/s41467-021-21217-0</a>.</li>
											<li>Rozenblatt-Rosen O, <b>Shin JW</b>, Rood JE, Hupalowska A, HCA STWG, Regev A, Heyn H. Towards Generating a High-Quality Human Cell Atlas: from Benchmarks to Application. <b>Nat. Biotechnol</b> Jan 26, 2021 <a href="https://doi.org/10.1038/s41587-020-00812-4"> https://doi.org/10.1038/s41587-020-00812-4</a>.</li>
											<li>Ramilowski JA, Yip CW, Agrawal S, Chang JC, Ciani Y, Kulakovskiy IV, Mendez M, Ooi JLC, Ouyang JF, Parkinson N, Petri A, Roos L, Severin J, Yasuzawa K, Abugessaisa I, Akalin A, Antonov IV, Arner E, Bonetti A, Bono H, Borsari B, Brombacher F, Cameron CJ, Cannistraci CV, Cardenas R, Cardon M, Chang H, Dostie J, Ducoli L, Favorov A, Fort A, Garrido D, Gil N, Gimenez J, Guler R, Handoko L, Harshbarger J, Hasegawa A, Hasegawa Y, Hashimoto K, Hayatsu N, Heutink P, Hirose T, Imada EL, Itoh M, Kaczkowski B, Kanhere A, Kawabata E, Kawaji H, Kawashima T, Kelly ST, Kojima M, Kondo N, Koseki H, Kouno T, Kratz A, Kurowska-Stolarska M, Kwon ATJ, Leek J, Lennartsson A, Lizio M, López-Redondo F, Luginbühl J, Maeda S, Makeev VJ, Marchionni L, Medvedeva YA, Minoda A, Müller F, Muñoz-Aguirre M, Murata M, Nishiyori H, Nitta KR, Noguchi S, Noro Y, Nurtdinov R, Okazaki Y, Orlando V, Paquette D, Parr CJC, Rackham OJL, Rizzu P, Sánchez Martinez DF, Sandelin A, Sanjana P, Semple CAM, Shibayama Y, Sivaraman DM, Suzuki T, Szumowski SC, Tagami M, Taylor MS, Terao C, Thodberg M, Thongjuea S, Tripathi V, Ulitsky I, Verardo R, Vorontsov IE, Yamamoto C, Young RS, Baillie JK, Forrest ARR, Guigó R, Hoffman MM, Hon CC, Kasukawa T, Kauppinen S, Kere J, Lenhard B, Schneider C, Suzuki H, Yagi K, de Hoon MJL, <b>Shin JW</b>, Carninci P. Functional Annotation of Human Long Non-Coding RNAs via Molecular Phenotyping. <b>Genome Res.</b> 2020. 30: 1060-1072.<a href="https://doi.org/10.1101/gr.254219.119">https://doi.org/10.1101/gr.254219.119</a>.</li>
											<li>Hashimoto K, Kouno T, Ikawa T, Hayatsu N, Miyajima Y, Yabukami H, Terooatea T, Sasaki T, Suzuki T, Valentine M, Pascarella G, Okazaki Y, Suzuki H, <b>Shin JW</b>, Minoda A, Taniuchi I, Okano H, Arai Y, Hirose N, Carninci P. Single-cell transcriptomics reveals expansion of cytotoxic CD4 T cells in supercentenarians. <b>Proc Natl Acad Sci U S A</b>. 2019 Nov 26;116(48):24242-24251. Epub 2019 Nov 12.<a href="https://doi.org/10.1073/pnas.1907883116">https://doi.org/10.1073/pnas.1907883116</a>.</li>
											<li>Kouno T, Moody J, Kwon A, Shibayama Y, Kato S, Huang Y, Böttcher M, Motakis E, Mendez M, Severin J, Luginbühl J, Abugessaisa I, Hasegawa A, Takizawa S, Arakawa T, Furuno M, Ramalingam N, West J, Suzuki H, Kasukawa T, Lassmann T, Hon CC, Arner E, Carninci P, Plessy C, <b>Shin JW</b>. C1 CAGE detects transcription start site and enhancer activity at single-cell resolution. <b>Nat. Commun</b> Article number:360 (2019).<a href="https://doi.org/10.1038/s41467-018-08126-5">https://doi.org/10.1038/s41467-018-08126-5</a>. </li>
											<li>de Rie D, Abugessaisa I, Alam T, Arner E, Arner P, Ashoor H, Åström G, Babina M, Bertin N, Burroughs AM, Carlisle AJ, Daub CO, Detmar M, Deviatiiarov R, Fort A, Gebhard C, Goldowitz D, Guhl S, Ha TJ, Harshbarger J, Hasegawa A, Hashimoto K, Herlyn M, Heutink P, Hitchens KJ, Hon CC, Huang E, Ishizu Y, Kai C, Kasukawa T, Klinken P, Lassmann T, Lecellier CH, Lee W, Lizio M, Makeev V, Mathelier A, Medvedeva YA, Mejhert N, Mungall CJ, Noma S, Ohshima M, Okada-Hatakeyama M, Persson H, Rizzu P, Roudnicky F, Sætrom P, Sato H, Severin J, <b>Shin JW</b>, Swoboda RK, Tarui H, Toyoda H, Vitting-Seerup K, Winteringham L, Yamaguchi Y, Yasuzawa K, Yoneda M, Yumoto N, Zabierowski S, Zhang PG, Wells CA, Summers KM, Kawaji H, Sandelin A, Rehli M; FANTOM Consortium, Hayashizaki Y, Carninci P, Forrest ARR, de Hoon MJL. An integrated expression atlas of miRNAs and their promoters in human and mouse. <b>Nat Biotechnol.</b> 2017 Sep;35(9):872-878. <a href="https://doi.org/10.1038/nbt.3947">https://doi.org/10.1038/nbt.3947</a></li>
											<li>Rackham OJ, Firas J, Fang H, Oates ME, Holmes ML, Knaupp AS; FANTOM Consortium, Suzuki H, Nefzger CM, Daub CO, <b>Shin JW</b>, Petretto E, Forrest AR, Hayashizaki Y, Polo JM, Gough J. A predictive computational framework for direct reprogramming between human cell types. <b>Nat Genet.</b> 2016 Jan 18.<a href="https://doi.org/10.1038/ng.3487">https://doi.org/10.1038/ng.3487</a></li>
											<li>FANTOM Consortium and the RIKEN PMI and CLST (DGT), Forrest AR, Kawaji H, Rehli M, Baillie JK, de Hoon MJ, Haberle V, Lassmann T, Kulakovskiy IV, Lizio M, Itoh M, Andersson R, Mungall CJ, Meehan TF, Schmeier S, Bertin N, Jørgensen M, Dimont E, Arner E, Schmidl C, Schaefer U, Medvedeva YA, Plessy C, Vitezic M, Severin J, Semple C, Ishizu Y, Young RS, Francescatto M, Alam I, Albanese D, Altschuler GM, Arakawa T, Archer JA, Arner P, Babina M, Rennie S, Balwierz PJ, Beckhouse AG, Pradhan-Bhatt S, Blake JA, Blumenthal A, Bodega B, Bonetti A, Briggs J, Brombacher F, Burroughs AM, Califano A, Cannistraci CV, Carbajo D, Chen Y, Chierici M, Ciani Y, Clevers HC, Dalla E, Davis CA, Detmar M, Diehl AD, Dohi T, Drabløs F, Edge AS, Edinger M, Ekwall K, Endoh M, Enomoto H, Fagiolini M, Fairbairn L, Fang H, Farach-Carson MC, Faulkner GJ, Favorov AV, Fisher ME, Frith MC, Fujita R, Fukuda S, Furlanello C, Furino M, Furusawa J, Geijtenbeek TB, Gibson AP, Gingeras T, Goldowitz D, Gough J, Guhl S, Guler R, Gustincich S, Ha TJ, Hamaguchi M, Hara M, Harbers M, Harshbarger J, Hasegawa A, Hasegawa Y, Hashimoto T, Herlyn M, Hitchens KJ, Ho Sui SJ, Hofmann OM, Hoof I, Hori F, Huminiecki L, Iida K, Ikawa T, Jankovic BR, Jia H, Joshi A, Jurman G, Kaczkowski B, Kai C, Kaida K, Kaiho A, Kajiyama K, Kanamori-Katayama M, Kasianov AS, Kasukawa T, Katayama S, Kato S, Kawaguchi S, Kawamoto H, Kawamura YI, Kawashima T, Kempfle JS, Kenna TJ, Kere J, Khachigian LM, Kitamura T, Klinken SP, Knox AJ, Kojima M, Kojima S, Kondo N, Koseki H, Koyasu S, Krampitz S, Kubosaki A, Kwon AT, Laros JF, Lee W, Lennartsson A, Li K, Lilje B, Lipovich L, Mackay-Sim A, Manabe R, Mar JC, Marchand B, Mathelier A, Mejhert N, Meynert A, Mizuno Y, de Lima Morais DA, Morikawa H, Morimoto M, Moro K, Motakis E, Motohashi H, Mummery CL, Murata M, Nagao-Sato S, Nakachi Y, Nakahara F, Nakamura T, Nakamura Y, Nakazato K, van Nimwegen E, Ninomiya N, Nishiyori H, Noma S, Noma S, Noazaki T, Ogishima S, Ohkura N, Ohimiya H, Ohno H, Ohshima M, Okada-Hatakeyama M, Okazaki Y, Orlando V, Ovchinnikov DA, Pain A, Passier R, Patrikakis M, Persson H, Piazza S, Prendergast JG, Rackham OJ, Ramilowski JA, Rashid M, Ravasi T, Rizzu P, Roncador M, Roy S, Rye MB, Saijyo E, Sajantila A, Saka A, Sakaguchi S, Sakai M, Sato H, Savvi S, Saxena A, Schneider C, Schultes EA, Schulze-Tanzil GG, Schwegmann A, Sengstag T, Sheng G, Shimoji H, Shimoni Y, <b>Shin JW</b>, Simon C, Sugiyama D, Sugiyama T, Suzuki M, Suzuki N, Swoboda RK, 't Hoen PA, Tagami M, Takahashi N, Takai J, Tanaka H, Tatsukawa H, Tatum Z, Thompson M, Toyodo H, Toyoda T, Valen E, van de Wetering M, van den Berg LM, Verado R, Vijayan D, Vorontsov IE, Wasserman WW, Watanabe S, Wells CA, Winteringham LN, Wolvetang E, Wood EJ, Yamaguchi Y, Yamamoto M, Yoneda M, Yonekura Y, Yoshida S, Zabierowski SE, Zhang PG, Zhao X, Zucchelli S, Summers KM, Suzuki H, Daub CO, Kawai J, Heutink P, Hide W, Freeman TC, Lenhard B, Bajic VB, Taylor MS, Makeev VJ, Sandelin A, Hume DA, Carninci P, Hayashizaki Y. A promoter level mammalian expression atlas. <b>Nature.</b> 2014 Mar 27;507(7493):462-70.<a href="https://doi.org/10.1038/nature13182">https://doi.org/10.1038/nature13182</a></li>
											<li>Kouno T, de Hoon M, Mar JC, Tomaru Y, Kawano M, Carninci P, Suzuki H, Hayashizaki Y, <b>Shin JW</b>. Temporal dynamics and transcriptional control using single cell analysis. <b>Genome Biol.</b> 2013 Oct 24;14(10):R118.<a href="https://doi.org/10.1186/gb-2013-14-10-r118">https://doi.org/10.1186/gb-2013-14-10-r118</a></li>
											<li>Hong YK, Foreman K, <b>Shin JW</b>, Hirakawa S, Curry CL, Sage DR, Libermann T, Dezube BJ, Fingeroth JD, Detmar M. Lymphatic reprogramming of blood vascular endothelium by Kaposi sarcoma-associated herpesvirus. <b>Nat Genet.</b> 2004 Jul;36(7):683-5.<a href="https://doi.org/10.1038/ng1383">https://doi.org/10.1038/ng1383</a></li>
										</ol>
								
							</section>

					</div>

				<!-- Footer -->
					<footer id="footer">
						<section>
							<h2> Genome Institute of Singapore </h2>
							<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d3988.7871326797167!2d103.79045891475396!3d1.3026845990498803!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x31da1a46a88725cd%3A0x9d27cec9775be342!2sGenome%20Institute%20of%20Singapore%20(GIS)!5e0!3m2!1sen!2ssg!4v1656919019429!5m2!1sen!2ssg" width="400" height="300" style="border:0;" allowfullscreen="" loading="lazy" referrerpolicy="no-referrer-when-downgrade"></iframe>
							
						</section>
						<section class="split contact">
							<section class="alt">
								<h3>Address</h3>
								<p>Genome Institute of Singapore (GIS)<br />
								60 Biopolis St, Singapore 138672</p>
							</section>
							<section>
								<h3>Phone</h3>
								<p>(065) 6808-8281</p>
							</section>
							<section>
								<h3>Email</h3>
								<p>jay_shin (at) gis (dot) a-star.edu.sg</p>
							</section>
							<section>
								<h3>Admin Email</h3>
								<p>han_lu (at) gis (dot) a-star.edu.sg</p>
								
							</section>
							<section>
								<h3>Social</h3>
								<ul class="icons alt">
									<li><a href="https://twitter.com/jayshin" class="icon brands alt fa-twitter"><span class="label">Twitter</span></a></li>
																	</ul>
							</section>
						</section>
					</footer>

				<!-- Copyright -->
					<div id="copyright">
						<ul><li>&copy; Untitled</li><li>Design: <a href="https://html5up.net">HTML5 UP</a></li></ul>
					</div>

			</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.scrollex.min.js"></script>
			<script src="assets/js/jquery.scrolly.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
